index.html

```diff
@@ -0,0 +1,157 @@
+        <p class="text-title">Tonic</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+
+      <div class="card">
+        <img src="./images/4.png" alt="">
+        <p class="text-title">Multi-post<br>stories</p>
+        <div class="experience">
+        <span class="subtitle">CANOPY</span>
+        <ul class="block">
+          <li>&bull; Back End Dev</li>
+          <li>&bull; 2015</li>
+        </ul>
+        </div>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <div class="but"><button class="button">See project</button></div>
+      </div>
+      
+
+      <section class="about-myself">
+        <div class="introduction">
+          <h1>About<br>Myself</h1>
+          <p class="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
+          <a href="#" class="connect">LET'S CONNECT</a><br><br>
+           <div class="socials">
+           <ul>
+            <li><a href=""><img src="./images/github.png" alt=""></a></li>
+            <li><a href=""><img src="./images/linkedin.png" alt=""></a></li>
+            <li><a href=""><img src="./images/v.png" alt=""></a></li>
+            <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+            <li><a href=""><img src="./images/m.png" alt=""></a></li>
+           </ul><br>
+            <button class="resume">Get my resume</button> <br><br><br>
+           </div>
+           <div class="lang-skills">
+             <span class="lg">Languages</span>
+             <img class="down" src="./images/10.png" ></img>
+        </div>
+        <ul class="show-skills">
+          <li class="sl">
+            <img class="lang-img" src="./images/js.png" alt="javascript"> 
+            <span class="span">JavaScript</span>
+          </li>
+          <li class="sl">
+            <img class="lang-img" src="./images/html.png" alt="html"> 
+            <span class="span">HTML</span>
+          </li>
+          <li class="sl">
+            <img class="lang-img" src="./images/css.png" alt="css"> 
+            <span class="span">CSS</span>
+          </li>
+        </ul>
+        <div class="frame">
+         <h1 class="lg">Frameworks</h1>
```

Comment:
- [OPTIONAL] I would highly suggest that you only one `h1` tag in your document because it enhances your website's SEO, (search engines use `h1` tags to determine the content of the page)